more/free-programming-interactive-tutorials-ru.md

```diff
@@ -18,8 +19,14 @@
 * [Обучение git при помощи визуализации](https://learngitbranching.js.org/?locale=ru_RU)
 
 
+### JavaScript
+
+* [CodeCombat - игры с программированием для изучения Python и JavaScript](http://codecombat.com)
```

Comment:
Codecombat is already as global resource in "ProblemSets & Competitive Programming" 

https://github.com/EbookFoundation/free-programming-books/search?q=codecombat

HTTPS is always better than HTTP

Reply:
fixed